packages/chart/src/components/BarChart/components/BarChart.Vertical.tsx

```diff
@@ -144,7 +144,13 @@ export const BarChartVertical = () => {
                   <li class="tooltip-body ">${tooltipBody}</li>
                   ${additionalColTooltip ? '<li class="tooltip-body ">' + additionalColTooltip + '</li>' : ''}
                     </li></ul>`
-
+                  const { barHeight, isSuppressed, getBarY, getAbsentDataLabel } = getBarConfig({
```

Comment:
Why not return the absentDataLabel directly instead of returning a function?

Reply:
Fixed!

Reply:
seems like it's still returning a function?

Reply:
@joshlacey  you probably dont have latest changes?